README.md

```diff
@@ -69,12 +69,25 @@ Statick is a plugin-based tool with an explicit goal to support external, option
 
 ## Installation
 
-Statick requires Python 3.6+ to run, but can be used to analyze Python2 projects, among many other languages.
+Statick requires Python 3 to run, but can be used to analyze Python 2 projects, among many other languages.
 
-The recommended install method is
+The recommended install method is to create a Python virtual environment and install from there.
```

Comment:
```suggestion
The recommended install method is to create a Python virtual environment and install there.
```
I'm not sure if `from there` or `to there` is better, but I kinda like taking out that word and leaving it as `install there`.